js/neighbourhoods.js

```diff
@@ -175,4 +175,81 @@ const neighbourhoods = [
     facts: "The focal point of this inner-city neighbourhood is the very picturesque Trinity-Bellwoods Park, which spans the entire length of the neighbourhood. This park features a paved walking path that is highlighted by distinguished cast iron lamp poles and a lush greenery. In addition to Trinity-Bellwoods Park, this neighbourhood also features affordable Victorian houses, excellent shopping districts, and convenient access to major transportation routes for motorists and pedestrians.",
     isConvenient: true
   },
+
+  // Malik
+
+  {
+    name: "Yorkdale-Glen Park",
+    //id:0,
+    zipcode: "M6B",
+    walkScore: 84,
+    facts: "This neighbourhood is also home to Lawrence Heights, a community undergoing revitalization, as well as several popular amenities including the Yorkdale and Lawrence Square shopping centres Local parks include Caledonia Park, Wenderly Park, Glen Long Park, Joyce Park, Ranee Park and Yorkdale Park. Dante Alighieri Academy is also situated in Yorkdale-Glen Park as well as Beth Torah Congregation",
+    isConvenient: false
+  },
+  
+  {
+    name: "Englemount-Lawrence",
+    //id:0,
+    zipcode: "M6B",
+    walkScore: 62,
+    facts: "The Englemount-Lawrence neighbourhood, with a population of 22,100, according to the 2011 Statistics Canada data, is bounded by Hwy. 401 to the north, Allen Road to the west, Briar Hill Avenue to the south, and Bathurst Street to the east",
+    isConvenient: false
+  },
+
+  {
+    name: "Yonge-Eglinton",
+    //id:0,
+    zipcode: "M5P",
+    walkScore: 96,
+    facts: "Yonge and Eglinton boasts family-centred neighbourhoods alongside a growing condo culture. Most people have to go kilometres to a movies theatre, but Yonge-and-Eg residents can walk to two multiplexes. Comedy lovers can hit Absolute Comedy at Yuk Yuk’s former location, and the retail and dining options have multiplied",
+    isConvenient: true
+  },
+
+  {
+    name: "Yonge-Eglinton",
+    //id:0,
+    zipcode: "M4P",
+    walkScore: 98,
+    facts: "Yonge and Eglinton boasts family-centred neighbourhoods alongside a growing condo culture. Most people have to go kilometres to a movies theatre, but Yonge-and-Eg residents can walk to two multiplexes. Comedy lovers can hit Absolute Comedy at Yuk Yuk’s former location, and the retail and dining options have multiplied",
+    isConvenient: true
+  },
```

Comment:
You have 2 of the same and missing the id's which could be found [here](https://www.toronto.ca/city-government/data-research-maps/neighbourhoods-communities/neighbourhood-profiles/) @irtzmalik